@table can no longer be used with commands that require no arguments
since Texinfo 5.0. So, convert the list to @itemize and improve the
output/style of nearby code instead of blindly substituting

  @table @bullet
  @item
  ...
  @end table

with

  @table @asis
  @item @bullet{}
  ...
  @end table

--- doc/xlogmaster.texinfo.orig	2009-06-09 00:34:43 UTC
+++ doc/xlogmaster.texinfo
@@ -710,90 +710,48 @@ The current version is 1.6.0, yet to be 
 @cindex Obtaining Xlogmaster
 
 The source can be found via anonymous FTP at
-@ifinfo
-@file{ftp.gnu.org:/pub/gnu/}.
-@end ifinfo
-@ifhtml
-<A HREF="ftp://ftp.gnu.org/pub/gnu/">ftp.gnu.org:/pub/gnu/</A>
-@end ifhtml
-@iftex
-@file{ftp.gnu.org:/pub/gnu/}.
-@end iftex
+@uref{ftp://ftp.gnu.org/pub/gnu/}.
 
 @node Getting help, Regexp, Where, Introductory Questions
 @subsection Where can I get help for using the Xlogmaster ?
 @cindex Help
 
 If you installed everything correctly the easiest ways are probably
-@table @bullet
+
+@itemize @bullet
 @item
 the tutorial that should be in your info tree. Otherwise you can find it
 in the @file{doc} directory of the distribution
+
 @item
 the man page.
+
 @item
 executing the Xlogmaster via @code{xlogmaster --help}
-@end table
+@end itemize
+
+@noindent
 Other ways are to check out the home page at
-@table @bullet
-@ifinfo
-@file{http://www.gnu.org/software/xlogmaster/xlogmaster.html}
-@end ifinfo
-@ifhtml
-<A HREF="http://www.gnu.org/software/xlogmaster/xlogmaster.html">http://www.gnu.org/software/xlogmaster/xlogmaster.html</A>
-@end ifhtml
-@iftex
-@file{http://www.gnu.org/software/xlogmaster/xlogmaster.html}
-@end iftex
-@end table
+@uref{http://www.gnu.org/software/xlogmaster/xlogmaster.html}
+@*
 or the mailing lists
+
 @cindex mailing lists
-@table @bullet
+@itemize @bullet
 @item
-@ifinfo
-@file{beta-xlogmaster@@gnu.org}
-@end ifinfo
-@ifhtml
-<A HREF="mailto:beta-xlogmaster@@gnu.org">beta-xlogmaster@@gnu.org</A>
-@end ifhtml
-@iftex
-@file{beta-xlogmaster@@gnu.org}
-@end iftex
-@sp 1
+@email{beta-xlogmaster@@gnu.org}
+
 Subscription is done by sending a mail to@*
-@ifinfo
-@file{beta-xlogmaster-request@@gnu.org}
-@end ifinfo
-@ifhtml
-<A HREF="mailto:beta-xlogmaster-request@@gnu.org">beta-xlogmaster-request@@gnu.org</A>
-@end ifhtml
-@iftex
-@file{beta-xlogmaster-request@@gnu.org}
-@end iftex
+@email{beta-xlogmaster-request@@gnu.org}
 @* leave the @emph{subject} blank and enter @samp{subscribe} in the @emph{body}.
+
 @item
-@ifinfo
-@file{help-xlogmaster@@gnu.org}
-@end ifinfo
-@ifhtml
-<A HREF="mailto:help-xlogmaster@@gnu.org">help-xlogmaster@@gnu.org</A>
-@end ifhtml
-@iftex
-@file{help-xlogmaster@@gnu.org}
-@end iftex
-@sp 1
+@email{help-xlogmaster@@gnu.org}
+
 Subscription is done by sending a mail to@*
-@ifinfo
-@file{help-xlogmaster-request@@gnu.org}
-@end ifinfo
-@ifhtml
-<A HREF="mailto:help-xlogmaster-request@@gnu.org">help-xlogmaster-request@@gnu.org</A>
-@end ifhtml
-@iftex
-@file{help-xlogmaster-request@@gnu.org}
-@end iftex
+@email{help-xlogmaster-request@@gnu.org}
 @* leave the @emph{subject} blank and enter @samp{subscribe} in the @emph{body}.
-@end table
+@end itemize
 
 @node Regexp, , Getting help, Introductory Questions
 @subsection Regular Expressions
